deployment/copilot/createReplayer.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+
+# Wish list
+# * Allow adding additional security group ids for the Replayer service
+
+# Allow executing this script from any dir
+script_abs_path=$(readlink -f "$0")
+script_dir_abs_path=$(dirname "$script_abs_path")
+cd $script_dir_abs_path
+
+usage() {
+  echo ""
+  echo "Create and Deploy Copilot Replayer service"
+  echo ""
+  echo "Options:"
+  echo "  --id*                                 [string] The unique ID to give this particular Replayer service, will be used in service naming (e.g. traffic-replayer-ID)"
+  echo "  --target-uri*                         [string] The URI of the target cluster that captured requests will be replayed to (e.g. https://my-target-cluster.com:443)"
+  echo "  --kafka-group-id                      [string, default: logging-group-ID] The Kafka consumer group ID that will decide which consumer group the Replayer will join and consume from"
+  echo "  --extra-args                          [string, default: --auth-header-user-and-secret MIGRATION_DOMAIN_USER_AND_SECRET_ARN] Extra arguments to provide to the Replayer command"
+  echo "  --delete-id                           [string, default: null] Delete the Replayer directory with the given ID (e.g. traffic-replayer-ID) and remove the Copilot service"
+  echo "  --copilot-app-name                    [string, default: migration-copilot] Specify the Copilot application name to use for deployment"
+  echo "  --skip-copilot-init                   Skip one-time Copilot initialization of Replayer service"
+  echo "  -r, --region                          [string, default: us-east-1] Specify the AWS region to deploy the CloudFormation stack and resources."
+  echo "  -s, --stage                           [string, default: dev] Specify the stage name to associate with the deployed resources"
+  echo ""
+  echo "* required"
+  exit 1
+}
+
+ID=""
+TARGET_URI=""
+KAFKA_GROUP_ID=""
+EXTRA_ARGS=""
+ID_TO_DELETE=""
+COPILOT_APP_NAME=migration-copilot
+SKIP_COPILOT_INIT=false
+REGION=us-east-1
+STAGE=dev
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    --id)
+      ID="$2"
+      shift # past argument
+      shift # past value
+      ;;
+    --target-uri)
+      TARGET_URI="$2"
+      shift # past argument
+      shift # past value
+      ;;
+    --kafka-group-id)
+      KAFKA_GROUP_ID="$2"
+      shift # past argument
+      shift # past value
+      ;;
+    --extra-args)
+      EXTRA_ARGS="$2"
+      shift # past argument
+      shift # past value
+      ;;
```

Comment:
I'm trying to keep this structure simple and easily predictable right now, but agree that this may be something users would prefer and we could change